include/acl_util.h

```diff
@@ -115,6 +115,15 @@ void acl_dump_mem(cl_mem mem);
 #define acl_dump_mem(x)
 #endif
 /////////////////////
+#define RETURN(ret)                                                            \
```

Comment:
Should I add the ACL prefix to all of them or for the new ones only? The RETURN macro was only added because it made it easy to change the code in bulk.

Reply:
Hi Eric, as we discussed offline, I don't think it's necessary to keep `RETURN` or `RETURN_VOID macro. Especially we will want to name them as ACL_RETURN, the user will wonder what it does differently from the standard C++ return. I agree with Peter, you should add ACL prefix for the new Macros that you created. Regarding the other existing runtime Macro, I think it's OK to keep those as is to limit the scope of the change. You would need to test the whole flow in case other component(e.g. the compiler) is using the existing Macro.

Reply:
Should I still update some of the newly added macros? I removed existing ones RETURN and RETURN_VOID. Other macros adhere to the code format from existing macros. I just added there non-unlocking versions?

Reply:
Do you mean adding ACL prefix for the newly added macros? I think yes, ideally we want to have ACL prefix for all of the existing macros too, but that can be done later because that gonna require more testing.